pages/common/katana.md

```diff
@@ -0,0 +1,28 @@
+# katana
+
+> A next-generation crawling and spidering tool.
+> More information: <https://github.com/projectdiscovery/katana>.
+
+- Crawl a [u]RL or a list of URLs:
+
+`katana -list {{https://example.com,https://google.com,...}}`
+
+- Crawl using [h]ead[l]ess mode using Chromium:
```

Comment:
```suggestion
- Crawl using headless mode using Chromium:
```

IMO it is non-standard that short options can be more than one character, so I'd just leave it out.

Reply:
@gutjuri This practice of using multi chars for short options exists in other places, e.g. `nuclei`, `httpx`

Reply:
It's a different case there, it's because they are for abbreviated options. We only use this when the long options is abbreviated or when there is a long option with these letters.

Reply:
I am finding it hard to follow. I think we should clarify the style guide. Anyway, I will accept the suggestion by gutjuri.